test-dts/component.test-d.ts

```diff
@@ -476,3 +478,12 @@ describe('class', () => {
 
   expectType<number>(props.foo)
 })
+
+describe('getCurrentInstance', () => {
```

Comment:
Do you mind moving this to another file pls? 
https://github.com/vuejs/core/blob/main/test-dts/componentTypeExtensions.test-d.tsx